src/Http/Controllers/Admin/TranslationsController.php

```diff
@@ -67,4 +73,196 @@ public function update(UpdateTranslation $request, Translation $translation)
         return redirect('admin/translation');
     }
 
+    public function export(UpdateTranslation $request)
+    {
+        return Excel::download(new TranslationsExport($request), 'translations.xlsx');
+    }
+
+    public function import(ImportTranslation $request) // FIXME create separate request class
+    {
+        if ($request->hasFile('fileImport')){
```

Comment:
Here is missing some response if file is missing. Some 409 error code or something.